.github/scripts/update-releases.py

```diff
@@ -116,6 +131,78 @@ def process_released_themes(overwrite=False, verbose=False):
     return designers
 
 
+def get_theme_download_count_preferring_previous(template, theme_downloads, current_name):
+    previous_download_count = get_previous_download_count_or_none(template, current_name)
+    if previous_download_count:
+        return previous_download_count
+
+    return get_theme_current_download_count(theme_downloads, current_name)
+
+
+def get_theme_current_download_count(theme_downloads, current_name):
+    return theme_downloads[current_name]["download"]
+
+
+def get_previous_download_count_or_none(template, current_name):
```

Comment:
Ah - This should have `theme` in the name, for consistency.. I don't mind whether to do it now or later...

Reply:
Sounds good to me, some of these might also make more sense in `themes.py`

Reply:
Oh thank you, that's a really good idea... I hadn't noticed that being used and so forgot about it...
I'm happy to move these to that file before finishing this PR - as I was conspicuous that a update-releases.py` had become quite long...

Reply:
D'oh - I can now see where themes.py is being used...

Reply:
@argenos Thanks for the suggestion - I've improved the function name, and moved a load of stuff the themes.py - I like it...

How do you feel about it now?